### DISCUSSION TITLE
PSAT cli tool result less cookie than extension

### DISCUSSION THREAD

**Discussion by Joeyindigital** [GoogleChromeLabs/ps-analysis-tool#640] [Support Forum] [answered]
Run test on single url and sitemap the result for total cookies and blocked cookies show massive different. 
The result from cli tool. 
![image](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/166688925/b9ffa94f-7631-4518-8301-f89d0752d808)

And the extension for single url. ( CDP enabled)
![image](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/166688925/8ae6ecbb-f120-42e4-850e-1c0dfd8216a5)

Is there any reason that cli tool show much lower than extension?

**Comment by milindmore22** [comment, marked as answer]
Hello @Joeyindigital 

Thanks for getting in touch!
The difference you’re seeing between the CLI and PSAT extension results might be due to things such as the environment, duration, user Interaction as described on our [wiki page](https://github.com/GoogleChromeLabs/ps-analysis-tool/wiki/PSAT-Command-Line-Interface#discrepancy-between-cli-and-extension).

Additionally, using a regular Chrome profile can cause issues because cookies remains, such as [orphaned cookies](https://github.com/GoogleChromeLabs/ps-analysis-tool/wiki/Cookies-Table#orphaned-cookies) and [unmapped cookies](https://github.com/GoogleChromeLabs/ps-analysis-tool/wiki/Cookies-Table#unmapped-cookies).

To avoid this, we recommend creating a separate Chrome profile specifically for using the PSAT Chrome extension. This keeps things clean and consistent.

For instructions on setting up an evaluation environment, please check our [wiki page](https://github.com/GoogleChromeLabs/ps-analysis-tool/wiki/Evaluation-Environment)

**Comment by milindmore22** [comment]
@Joeyindigital I will be converting this issue to support forum discussion, For future reference, please [create support discussion](https://github.com/GoogleChromeLabs/ps-analysis-tool/discussions/new?category=support-forum) if you have any question about the PSAT.
